p2p/src/main/kotlin/maru/p2p/NoopP2PNetwork.kt

```diff
@@ -0,0 +1,30 @@
+/*
+   Copyright 2025 Consensys Software Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package maru.p2p
+
+import tech.pegasys.teku.infrastructure.async.SafeFuture
+
+object NoopP2PNetwork : P2PNetwork {
```

Comment:
```suggestion
object NoOpP2PNetwork : P2PNetwork {
```
NoOp -> No Operation

Reply:
[Noop](https://en.wikipedia.org/wiki/NOP_(code)) is widely known

23 M entries of [Noop](https://github.com/search?q=Noop&type=code) vs 53 K of [NoOperation](https://github.com/search?q=Nooperation&type=code) on Github

Reply:
I meant the casing for  NoopP2PNetwork should be changed to No**O**pP2PNetwork with a capital O